scripts/test-integration-aws.sh

```diff
@@ -8,4 +8,5 @@ set -euo pipefail
 #   Run integration tests for AWS components only.
 
 docker-compose up -d dependencies-aws
+docker-compose up -d dependencies-elasticsearch
```

Comment:
You can join this with the previous line and bring both up at once. Also, why is this needed for the AWS tests? Is it because the elasticsearch sink is dual-mode?

Reply:
Oh, indeed, I can specify both services.
I added this service because `aws-integration-tests` have dependency `es-integration-tests`, but without service tests will fail. I think right thing will remove service and dependency, which was probably added to test aws credentials.